cmd/legacy_main.go

```diff
@@ -87,13 +89,24 @@ func registerTerminatingSignalHandler(mountPoint string, c *cfg.Config) {
 
 func getUserAgent(appName string, config string) string {
```

Comment:
```suggestion
func getUserAgent(appName string, config string, mountId string) string {
```

I feel, we should make caller's responsibility to pass the unique ID. Other wise, different invocation with the same parameter will return different user-agent string.